src/main/features/response/form/models/statement-of-means/bankAccountRow.ts

```diff
@@ -0,0 +1,62 @@
+import { IsDefined, IsIn, ValidateIf } from 'class-validator'
+
+import { ValidationErrors as GlobalValidationErrors } from 'forms/validation/validationErrors'
+import { MultiRowFormItem } from 'forms/models/multiRowFormItem'
+import { Fractions } from 'forms/validation/validators/fractions'
+import * as toBoolean from 'to-boolean'
+import { toNumberOrUndefined } from 'common/utils/numericUtils'
+import { BankAccountType } from 'response/form/models/statement-of-means/bankAccountType'
+
+export class ValidationErrors {
+  static readonly TYPE_OF_ACCOUNT_REQUIRED: string = 'Select a type of account'
+}
+
+export class BankAccountRow extends MultiRowFormItem {
+
+  @ValidateIf(o => o.typeOfAccount !== undefined || o.isJoint !== undefined || o.balance !== undefined)
```

Comment:
Maybe worth adding `isEmpty` method that checks `o.typeOfAccount !== undefined || o.isJoint !== undefined || o.balance !== undefined` and then use that one in `@ValidateIf`

Thank could make it more readable and remove repetition.

Reply:
ok